DashAI/front/src/components/experiments/ConfigureModelsStep.jsx

```diff
@@ -0,0 +1,182 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import { Button, Grid, MenuItem, TextField, Typography } from "@mui/material";
+import { AddCircleOutline as AddIcon } from "@mui/icons-material";
+import { getModelSchema as getModelSchemaRequest } from "../../api/oldEndpoints";
+import { useSnackbar } from "notistack";
+import ModelsTable from "./ModelsTable";
+import { getFullDefaultValues } from "../../utils/values";
+
+/**
+ * Step of the experiment modal: add models to the experiment and configure its parameters
+ * @param {object} newExp object that contains the Experiment Modal state
+ * @param {function} setNewExp updates the Eperimento Modal state (newExp)
+ * @param {function} setNextEnabled function to enable or disable the "Next" button in the modal
+ */
+function ConfigureModelsStep({ newExp, setNewExp, setNextEnabled }) {
+  const { enqueueSnackbar } = useSnackbar();
+  const [modelNickname, setModelNickname] = useState("");
+  const [selectedModel, setSelectedModel] = useState("");
+  // const [modelsInExperiment, setModelsInExperiment] = useState([]);
+
+  const compatibleModels = [
+    "KNeighborsClassifier",
+    "RandomForestClassifier",
+    "SVC",
+    "numericalwrapperfortext",
+    "tcTransformerEngSpa",
+  ];
+
+  // width for model nickname and model type textfields
+  const textFieldWidth = "32vw";
+
+  // gets a new id for a model by adding 1 to the max id of the current models in the experiment
```

Comment:
Each model will map to a unique run right?
Then, why not to use an [UUID](https://interactivechaos.com/es/wiki/identificador-unico-universal-uuid)?
Here are a simple reference of this: [UUID in JS - Stackverflow](https://stackoverflow.com/questions/105034/how-do-i-create-a-guid-uuid)